submissions/2020-12-18-OpenFF-Theory-Benchmarking-Set-v1.0/compute.json

```diff
@@ -0,0 +1,95 @@
+{
+  "qc_specifications": {
+    "WB97X-D3BJ": {
+      "method": "WB97X-D3BJ",
+      "basis": "DZVP",
+      "program": "psi4",
+      "spec_name": "WB97X-D3BJ",
+      "spec_description": "A torsiondrive dataset for benchmarking WB97X-D3BJ",
+      "store_wavefunction": "none",
+      "implicit_solvent": null
+    },
+    "PW6B95-D3BJ": {
+      "method": "PW6B95-D3BJ",
+      "basis": "DZVP",
+      "program": "psi4",
+      "spec_name": "PW6B95-D3BJ",
+      "spec_description": "A torsiondrive dataset for benchmarking PW6B95-D3BJ",
+      "store_wavefunction": "none",
+      "implicit_solvent": null
+    },
+    "B3LYP-D3MBJ/DZVP": {
+      "method": "B3LYP-D3MBJ",
+      "basis": "DZVP",
+      "program": "psi4",
+      "spec_name": "B3LYP-D3MBJ/DZVP",
+      "spec_description": "A torsiondrive dataset for benchmarking B3LYP-D3MBJ/DZVP ",
+      "store_wavefunction": "none",
+      "implicit_solvent": null
+    }
+  },
+  "dataset_name": "OpenFF Theory Benchmarking Set v1.0",
+  "dataset_tagline": "Torsiondrives for theory benchmarking",
+  "dataset_type": "TorsiondriveDataset",
+  "maxiter": 200,
+  "driver": "gradient",
+  "scf_properties": [
+    "dipole",
+    "quadrupole",
+    "wiberg_lowdin_indices",
+    "mayer_indices"
+  ],
+  "priority": "normal",
+  "description": "A torsiondrive dataset for theory benchmarking",
+  "dataset_tags": [
+    "openff"
+  ],
+  "compute_tag": "openff",
+  "metadata": {
+    "submitter": "pavankum",
+    "creation_date": "2021-06-16",
+    "collection_type": "TorsiondriveDataset",
+    "dataset_name": "OpenFF Theory Benchmarking Set v1.0",
+    "short_description": "Torsiondrives for theory benchmarking",
+    "long_description_url": "https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2020-12-18-OpenFF-Theory-Benchmarking-Set-v1.0",
+    "long_description": "Adding wB97X-D3BJ compute spec to the torsiondrive dataset for theory benchmarking",
+    "elements": [
+      "O",
+      "N",
+      "F",
+      "P",
+      "H",
+      "S",
+      "C",
+      "Cl"
+    ]
+  },
+  "provenance": {
+    "qcsubmit": "0.2.1",
+    "openforcefield": "0.8.4",
+    "rdkit": "2021.03.3"
+  },
+  "dataset": {},
+  "filtered_molecules": {},
+  "optimization_procedure": {
+    "program": "geometric",
+    "coordsys": "tric",
```

Comment:
I assume we want to stick with `tric` for the `coordsys`, since that's what the rest of the original dataset specs use?

Reply:
Thank you @dotsdl , I changed to dlc for this set after your recommendation, I guess it should not be an issue as long as I get converged runs.